src/Moryx.CommandCenter.Web/src/modules/container/Notifications.tsx

```diff
@@ -40,19 +75,27 @@ export class Notifications extends React.Component<NotificationsPropModel> {
                         </TableRow>
                     </TableHead>
                     <TableBody>
-                        { (messages.length > 0)
-                            ? (messages.map((message, idx) => (
-
+                        { (paginatedMessages.length > 0)
+                            ? (paginatedMessages.map((message, idx) => (
                                 <NotificationRow message={message}/>
                             )))
                             : <TableRow>
                                 <TableCell align="center" colSpan={4}>No notifications</TableCell>
                             </TableRow>
                         }
                     </TableBody>
+                    <TableFooter>
+                        <TablePagination
+                            count={messages.length}
+                            page={page}
+                            onPageChange={handlePageChange}
+                            rowsPerPage={rowsPerPage}
+                            onRowsPerPageChange={handleChangeRowsPerPage}
+                            rowsPerPageOptions={[5, 10, 25]}
```

Comment:
Where did you get these values from? I'd vote for something like 25, 50, 100, as 5 and 10 seem to be quite low.